Backend/package.json

```diff
@@ -12,12 +12,14 @@
   "license": "ISC",
   "dependencies": {
     "express": "^4.17.3",
+    "latest": "^0.2.0",
     "luxon": "^2.3.0",
     "socket.io": "^2.2.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.17.3",
     "@babel/core": "^7.17.5",
-    "@babel/preset-env": "^7.16.11"
+    "@babel/preset-env": "^7.16.11",
+    "prettier": "2.7.1"
```

Comment:
Sorry for the confusion. The Prettier comment in the README was more for individual use (e.g. installing Prettier for VSCode) rather than for this project specifically. If you could run `npm uninstall prettier` in Backend to remove the package, that'd be awesome.